README.md

```diff
@@ -6,7 +6,7 @@
 | Download | [![NuGet Downloads](http://img.shields.io/nuget/dt/paramore.brighter.svg)](https://www.nuget.org/packages/Paramore.Brighter/) |
 | Documentation  |  [Technical Documentation](https://brightercommand.gitbook.io/paramore-brighter-documentation/)  |
 | Source  |https://github.com/BrighterCommand/Brighter |
-| Keywords  |task queue, job queue, asynchronous, async, rabbitmq, amqp, sqs, sns, kakfa, redis, c#, command, command dispatcher, command  processor, queue, distributed |
+| Keywords  |task queue, job queue, asynchronous, async, rabbitmq, amqp, sqs, sns, kafka, redis, c#, command, command dispatcher, command  processor, queue, distributed |
```

Comment:
I spell kafka like this all the time, and looking at this recent PR title, old habits die hard 😄 
https://github.com/BrighterCommand/Brighter/pull/3014